content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -304,7 +304,13 @@ mvn -Dset.changelist \
 
 == Troubleshooting
 
-Check that `MAVEN_TOKEN` and `MAVEN_USERNAME` appear under Repository secrets and that your pull request has a label indicating it was of interest to users (e.g., enhancement rather than chore).
+Check that `MAVEN_TOKEN` and `MAVEN_USERNAME` appear under Repository secrets.
+
+=== No release has been triggered after merging a pull request
```

Comment:
Can these labels be added afterward then rerun the CD action? Asking as I'm not sure.

Reply:
they can be yes

Reply:
That may cause Release Drafter to rearrange release notes, but if you manually trigger the CD action it will publish regardless of any labels.

Reply:
Or if you add an interesting label to a PR that was already merged, and then merge _another_ PR (even with an uninteresting label), then you will get a release, since the workflow will see that there has been at least interesting PR since the last release.

Reply:
(It would be great if someone could volunteer to handle https://github.com/jenkins-infra/interesting-category-action/issues/1 which is probably not a lot of work, just picking an existing action and testing it on https://github.com/jenkinsci/jenkins-infra-test-plugin to make sure it behaves as expected.)

Reply:
Thanks for these complements, added in https://github.com/jenkins-infra/jenkins.io/pull/7686/commits/dcbc3ac887ae5b34d1cf27ead9bd78d9e42ef94d

I'll take a look at https://github.com/jenkins-infra/interesting-category-action/issues/1.